tasks/tests/tasks_tests/fixtures.py

```diff
@@ -8,3 +8,8 @@ def resources_dir():
     resources = Path(__file__).parent / "resources"
     assert resources.exists()
     return resources
+
+
+@pytest.fixture(autouse=True)
+def set_check_truststore_false(monkeypatch):
+    monkeypatch.setenv("RC_USE_TRUSTSTORE", "False")
```

Comment:
If by any means we'll change the env var, please update it here as well.

Make sure to put a comment above explaining what the line does, something like:
> "Disables truststore SSL injection during testing so we'd avoid unnecessary warnings given the missing dependency."